src/main/java/com/project/gamestore/controllers/RAWGController.java

```diff
@@ -62,20 +62,41 @@ public void processTitle(String title) {
 
     @GetMapping("/videogames/search")
     public List<VideoGame> ListGamesByName(@RequestParam String title) throws JsonProcessingException {
```

Comment:
Add page_size parameter and page number parameter. So the front end can navigate through pages.